src/helpers.js

```diff
@@ -240,20 +234,23 @@ export async function processESLintMessages(response, textEditor, showRule, work
         range = generateRange(textEditor, msgLine, msgCol)
       }
       ret = {
-        filePath,
-        type: severity === 1 ? 'Warning' : 'Error',
-        range
+        severity: severity === 1 ? 'warning' : 'error',
+        location: {
+          file: filePath,
+          position: range
+        }
       }
 
       if (showRule) {
-        const elName = ruleId ? 'a' : 'span'
-        const href = ruleId ? ` href="${ruleURI(ruleId).url}"` : ''
-        ret.html = `${escapeHTML(message)} (<${elName}${href}>${ruleId || 'Fatal'}</${elName}>)`
+        const url = ruleId ? ruleURI(ruleId).url : ''
+        // TODO: fetch rule Markdown from eslint
+        ret.url = url
+        ret.description = `${escapeHTML(message)} ([${ruleId || 'Fatal'}](${url}))`
```

Comment:
This should be `ret.excerpt`. `description` is only shown on expanding the message and is only intended for blocks of descriptive text, `excerpt` is the equivalent of the old `text/html` attribute.